tools/syz-cover/syz-cover.go

```diff
@@ -157,3 +168,44 @@ func loadModules(fname string) ([]host.KernelModule, error) {
 	}
 	return modules, nil
 }
+
+func loadModulesFromFile(cfg *mgrconfig.Config, fname string) ([]host.KernelModule, error) {
```

Comment:
We already do the parsing here: https://github.com/google/syzkaller/blob/master/pkg/host/machine_info_linux.go#L127

I'd export some `ParseLinuxModules` function from `pkg/host` and use it here. Let's not duplicate the code.

Reply:
done